src/lightning-stubs/inputField/inputField.js

```diff
@@ -10,7 +10,7 @@ export default class InputField extends LightningElement {
     @api dirty;
     @api disabled;
     @api fieldName;
-    @api readonly;
+    @api readOnly;
```

Comment:
Isn't this a breaking change?

Reply:
The change here to the stub is reflective of how the real component was modified. 

@Templarian @vbhargavmbis Can you two confirm this change to the public components API shape is expected?